nodes/index.adoc

```diff
@@ -17,6 +17,18 @@ Having stable and healthy nodes in a cluster is fundamental to the smooth functi
 In {product-title}, you can access, manage, and monitor a node through the `Node` object representing the node.
 Using the OpenShift CLI (`oc`) or the web console, you can perform the following operations on a node.
 
+Following are the components of a node responsible for maintaining the running of pods and providing the Kubernetes runtime environment.
+
+* *Container runtime*: The container runtime is the software that is responsible for running containers. Kubernetes supports several runtimes: Docker, containerd, cri-o, rktlet, and any implementation of the Kubernetes Container Runtime Interface (CRI).
```

Comment:
I would change this wording because there are more than several runtimes listed here. Maybe "Kubernetes supports runtimes such as Docker, containerd, cri-o, rklet, and any implementation of the Kubernetes Container Runtime Interface (CRI)."

Reply:
Addressed